apps/framework-cli/src/cli/settings.rs

```diff
@@ -34,11 +34,21 @@ impl Default for Features {
 #[derive(Deserialize, Debug)]
 pub struct Telemetry {
     pub enabled: bool,
+
+    #[serde(default = "default_moose_developer")]
```

Comment:
The `Default::default` of `bool` is `false`, I don't think we need this func.

Reply:
I will try

Reply:
You are right!